src/components/StreamSwitcherCard.vue

```diff
@@ -81,162 +59,107 @@ export default {
-  width: 250px;
-  min-width: 250px;
-  @include typeface(body, 4);
-  border: 1px solid RGB(var(--color-background));
-  background: transparent;
-  color: RGB(var(--color-background));
+  flex-wrap: wrap;
   cursor: pointer;
-
-  &.is-active {
-    background: RGB(var(--color-background));
-    color: RGB(var(--color-text));
-    &::after {
-      content: '';
-      position: absolute;
-      width: 0;
-      bottom: -16px;
-      height: 15px;
-      left: 30px;
-      border-left: 20px solid transparent;
-      border-right: 20px solid transparent;
-      border-top: 15px solid RGB(var(--color-text));
-      opacity: 0;
-      transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
-    }
+  border: 1px solid RGB(var(--color-text-inverse));
+  padding: var(--space-1) var(--space-2) var(--space-2);
+  height: 70px;
+  align-items: center;
+  width: fit-content;
+  max-width: 320px;
+  min-width: 170px;
+  margin-bottom: -20px; // hack to get around overflow scroll issue
+  transition: var(--animation-duration-slow) var(--animation-easing-incoming);
+  background-color: transparent;
+  color: RGB(var(--color-text-inverse));
+
+  @include media(">xlarge") {
+    padding: var(--space-2) var(--space-3);
+    height: 50px;
+    flex-wrap: nowrap;
   }
-}
-
-.stream-switcher-card .stream-switcher-card-station {
-  margin: var(--space-2);
-  font-weight: bold;
-  text-transform: uppercase;
-}
 
-.stream-switcher-card .stream-switcher-card-sound-animation {
-  position: absolute;
-  align-items: flex-end;
-  align-self: center;
-  display: flex;
-  justify-content: space-between;
-  flex-direction: row;
-  top: var(--space-2);
-  right: var(--space-2);
-  height: 16px;
-  width: 40px;
-
-  &.is-playing {
-    .sound-animation-bar {
-      animation: sound-bars 500ms linear 0ms infinite alternate;
-    }
-
-    .sound-animation-bar:nth-child(1) {
-      animation-delay: 0ms;
-    }
-
-    .sound-animation-bar:nth-child(2) {
-      animation-delay: 100ms;
-    }
-
-    .sound-animation-bar:nth-child(3) {
-      animation-delay: 200ms;
-    }
-
-    .sound-animation-bar:nth-child(4) {
-      animation-delay: 300ms;
-    }
+  // active caret
+  &::after {
+    content: '';
+    position: absolute;
+    width: 0;
+    bottom: -16px;
+    height: 15px;
+    left: 30px;
+    border-left: 20px solid transparent;
+    border-right: 20px solid transparent;
+    border-top: 15px solid RGB(var(--color-text-inverse));
+    opacity: 0;
+    -webkit-transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
+    -moz-transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
+    -ms-transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
+    -o-transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
+    transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
```

Comment:
it would be nice to get postcss and autoprefixer set up sometime

Reply:
Autoprefixing is included in Nuxt.js and is handled by autoprefixer - maybe I can get rid of this (I needed it for the Ember version)

Reply:
Also need to add a knob or something so people can toggle the active state on and off to preview the animation. I have an animation ticket and I will figure it out while working on that!